### ISSUE TITLE
Adaptive Feedback Regulator for Powered Lower-Limb Exoskeleton under Model Uncertainty. (arXiv:2104.11775v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#6347]
Adaptive Feedback Regulator for Powered Lower-Limb Exoskeleton under Model Uncertainty. (arXiv:2104.11775v1 [eess.SY])<br>
https://ift.tt/3xnUMmC<br>
<p>This paper presents a neural network (NN) based adaptive feedback regulator to ensure the lateral and longitudinal stability and regulate the desired walking velocity of a lower-limb exoskeleton under model uncertainty. The traditional model-based controllers for lower-limb exoskeletons often fail to stabilize the robot or accurately track the desired behaviors under model uncertainties or external disturbances. This paper proposes a neural network (NN) based online adaptive regulator that compensates for the unknown changes in model parameters and external disturbances by modifying the nominal joint trajectory. A gradient descent-based delta rule is implemented to update the weights of a single layer NN, which can be efficiently performed online by design. We demonstrate the performance of the presented regulator on ATALANTE, a fully actuated lower limb exoskeleton designed for paraplegic patients. The simulation results show that the proposed approach noticeably improves stability and the tracking performance of the system, despite significant changes in model parameters and large adversarial pushes.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
